pset-iterators.js

```diff
@@ -138,6 +172,12 @@ console.log('Problem 5:')
 
 // Add your code below this line
 
+const livingPresidents = presidents.filter(function (name) {
```

Comment:
@jacobkelman nice work, only recommendation is to use a more contextual parameter name for your callback, `president` would have made more sense than `name` - since each object (in the president's array) represents a president and not just the president's name